RELEASE.md

```diff
@@ -1,31 +1,31 @@
-# RELEASE NOTES for FV3: Summary
+# RELEASE NOTES for GFDL  FV3: Summary
 
-FV3-201912-public --- 10 January 2020
-Lucas Harris, GFDL <lucas.harris@noaa.gov>
+2020.02 --- 22 April 2020
 
-This version has been tested against the current GFDL AM4 phsyics
-and with release candidate 2020.02 of the FMS code at https://github.com/NOAA-GFDL/FMS
+This version has been tested within current GFDL Models (AM, CM, ESM, SPEAR, etc.) and requires the 2020.02 release of the [FMS infrastructure](https://github.com/NOAA-GFDL/FMS).
 
-Includes all of the features of the GFDL Release to EMC, as well as:
+Includes all of the features of the [201912 Public Release](https://github.com/NOAA-GFDL/GFDL_atmos_cubed_sphere/releases/tag/201912_public_release) which include:
 
-- Updated 2017 GFDL Microphysics (from S-J Lin and L Zhou included in GFSv15)
-- Updates for GFSv15 ICs (from T Black/J Abeles, EMC)
-- Updates to support new nesting capabilities in FMS (from Z Liang)
+- Updates to support new nesting capabilities in FMS
 - Re-written grid nesting code for efficiency and parallelization
 - Re-organized fv_eta for improved vertical level selection
-- 2018 Stand-alone regional capabilities (from T Black/J Abeles, EMC)
+- 2018 Stand-alone regional capabilities (from EMC)
 - Refactored model front-end (fv_control, fv_restart)
 - Support for point soundings
 - And other updates
 
-# Interface changes
+# Directory structure changes
 
-drivers: renamed 'fvGFS' directory to SHiELD
+***drivers/*** 
+  - renamed ***fvGFS*** to ***SHiELD***
+  - renamed ***coupled*** to ***GFDL***
```

Comment:
This happened at the beginning of the dev/gfdl branch.  It's not new to this release.

Reply:
@thomas-robinson I was thinking of those coming from the locked internal project and starting to use the publicly available repository.  Do you have suggested wording for that?

Reply:
- *coupled* is *GFDL*  (for internal GFDL users) (?) I'm not sure.  This is probably something that the liaison's should convey to the groups

Reply:
see update